projects/1.20/assets/enderite-cj-edition/enderite/lang/zh_cn.json

```
@@ -0,0 +1,113 @@
+{
+  "creativemodetab.enderite.enderite_tab": "末影合金",
+
+  "enchantment.enderite.night_vision": "夜视",
+  "enchantment.enderite.no_gravity": "失重",
+  "enchantment.enderite.void_floating": "虚空飘浮",
+  "enchantment.enderite.night_vision.desc": "给予玩家夜视效果",
+  "enchantment.enderite.no_gravity.desc": "赋予物品无重力",
+  "enchantment.enderite.void_floating.desc": "保护物品不受虚空伤害",
+
+  "item.enderite.ender_crystal_shard": "末影水晶碎片",
+  "item.enderite.ender_dust": "末影尘",
+  "item.enderite.raw_enderite": "末影合金粗矿",
+  "item.enderite.enderite_scrap": "末影合金碎片",
+  "item.enderite.enderite_ingot": "末影合金锭",
+  "item.enderite.enderite_nugget": "末影合金粒",
+
+  "item.enderite.enderite_helmet": "末影合金头盔",
+  "item.enderite.enderite_chestplate": "末影合金胸甲",
+  "item.enderite.enderite_elytra_chestplate": "末影合金鞘翅胸甲",
+  "item.enderite.enderite_leggings": "末影合金护腿",
+  "item.enderite.enderite_boots": "末影合金靴子",
+
+  "item.enderite.enderite_sword": "末影合金剑",
+  "item.enderite.enderite_pickaxe": "末影合金镐",
+  "item.enderite.enderite_axe": "末影合金斧",
+  "item.enderite.enderite_shovel": "末影合金铲",
+  "item.enderite.enderite_hoe": "末影合金锄",
+  "item.enderite.ender_pearl_staff": "末影珍珠杖",
+
+  "block.enderite.enderite_block": "末影合金块",
+  "block.enderite.enderite_ore": "末影合金矿",
+  "block.enderite.cracked_enderite_ore": "破碎的末影合金矿",
+  "block.enderite.raw_enderite_block": "粗末影合金块",
+  "block.enderite.end_respawn_anchor": "末地重生锚",
+  "block.enderite.hardened_end_stone": "压缩末地石",
+  "block.enderite.budding_ender_crystal": "末影水晶母岩",
+  "block.enderite.ender_crystal_block": "末影水晶矿石",
+  "block.enderite.ender_crystal_cluster": "末影水晶簇",
+  "block.enderite.large_ender_crystal_bud": "大型末影水晶芽",
+  "block.enderite.medium_ender_crystal_bud": "中型末影水晶芽",
+  "block.enderite.small_ender_crystal_bud": "小型末影水晶芽",
+
+  "tooltip.enderite.only_creative": "仅限创造模式",
+  "tooltip.enderite.enderite_sword_teleport": "将你传送到指针所选方块处",
+  "tooltip.enderite.enderite_sword_teleport_range": "方块范围",
+  "tooltip.enderite.enderite_sword_teleport_charges_info1": "想要充能，请在你的物品栏中放入珍珠，然后点击鼠标右键",
+  "tooltip.enderite.enderite_sword_teleport_charges_info2": "潜行+右键点击传送",
+  "tooltip.enderite.enderite_sword_teleport_charges": "珍珠充能",
+  "tooltip.enderite.enderite_sword_levitation": "赐予目标飘浮效果",
+  "tooltip.enderite.enderite_sword_health_boost": "攻击时短暂增加自身生命值",
+  "tooltip.enderite.enderite_pickaxe": "更快地摧毁末地石",
+  "tooltip.enderite.enderite_pickaxe_configurable": "你可以在 'data/forge/tags/blocks/end_stones.json'中配置被镐破坏得更快的方块",
```

```suggestion
  "tooltip.enderite.enderite_pickaxe_configurable": "你可以在'data/forge/tags/blocks/end_stones.json'中配置被镐破坏得更快的方块",
```
单引号的符号问题这里开一条，别resolve，需要讨论

---

不需要加空格
但是按照标点符号规范，引号推荐改为全角双引号
主要是这里是个路径，感觉应该放群里问一句